StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -228,9 +228,11 @@ extension EmbeddedPaymentElement: VerticalSavedPaymentMethodsViewControllerDeleg
             savedPaymentMethods: latestPaymentMethods
         )
 
-        // If there are still saved payment methods & the saved payment method was previously selected to presenting
+        // Select the selected payment method if we selected a payment method and have more than one saved PM
+        // Or if there are still saved payment methods & the saved payment method was previously selected to presenting
```

Comment:
// Select the *saved payment method if we selected a payment method
// or
//  there are still saved payment methods & the saved payment method was previously selected to presenting